src/main.js

```diff
@@ -244,3 +244,81 @@ let destinations = [
   ]
 
 
+  //visited and bucktelist lists
+  const visited = destinations.filter(item => item.visited === true);
+  const bucketlist = destinations.filter(item => item.visited === false);
+ 
+
+  //get list of urls
+  const getURL = (destinations) => {
+    let url= destinations.map(obj => obj.photo.valueOf());
+    return url;
+  }
+
+  //create elements for each city
+  const elements= destinations.forEach(destination => {
+
+    //picture element
+    const createImg = (url) => {
+      let figure = document.createElement('figure');
+    //img element
+      let img = document.createElement('img');
+      img.style.objectFit ='cover';
+      document.body.appendChild(figure);
+      figure.appendChild(img);
+    //img caption
+      let figcaption= document.createElement('figcaption');
+      figure.appendChild(figcaption);
+      figcaption.innerText= `${destinations.city}, ${destinations.country}`
```

Comment:
Here `destinations.city` does not work. It should be `destination.city`. The first is a list (array) the other is the destination itself (the object).

A tip to avoid this mistakes. Name lists like this `destinationList` `animalList` etc... that what you detect immediately if it's a list or not ;)